cmd/kubeadm/app/cmd/util/cmdutil.go

```diff
@@ -74,3 +74,13 @@ func FindExistingKubeConfig(file string) string {
 	rules.Precedence = append(rules.Precedence, kubeadmconstants.GetAdminKubeConfigPath())
 	return rules.GetDefaultFilename()
 }
+
+// SetKubernetesVersion gets the current Kubeadm version and sets it as KubeadmVersion in the config,
+// unless it's already set to a value different from the default.
+func SetKubernetesVersion(cfg *kubeadmapiv1beta1.InitConfiguration) {
```

Comment:
the function should be moved to this file instead:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/util/config/common.go

Reply:
I will remove this entirely from this commit and make a new PR for it's move to the above file.